modules/spa-s3-cloudfront/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+## Components PR [#948](https://github.com/cloudposse/terraform-aws-components/pull/948)
+
+The previous PR [#946](https://github.com/cloudposse/terraform-aws-components/pull/946) introduced the `var.lambda_runtime` input. Previously, the version of node in both submodules was hard-coded to be `nodejs12.x`. The default value should be the latest, recommended version. This PR sets that default to `nodejs20.x`.
+
+If you want to maintain the previous version of node, set `var.lambda_runtime` to `nodejs12.x`. Otherwise, this component will attempt to deploy the functions with runtime `nodejs20.x`.
```

Comment:
```suggestion
If you want to maintain the previous version of node, set `var.lambda_runtime` to `nodejs12.x`, though be aware that AWS deprecated that version on March 31, 2023, and lambdas using that environment may no longer work. Otherwise, this component will attempt to deploy the functions with runtime `nodejs20.x`.
```